router/batchrouter/asyncdestinationmanager/bing-ads/bingads.go

```diff
@@ -23,12 +24,17 @@ import (
 	"github.com/rudderlabs/rudder-server/utils/misc"
 )
 
+const pollUrl = "https://bulk.api.bingads.microsoft.com/Api/Advertiser/CampaignManagement/v13/BulkService.svc"
```

Comment:
this is service url right? not specific to poll right?

Reply:
removed